shelley/chain-and-ledger/formal-spec/utxo.tex

```diff
@@ -384,7 +384,7 @@ \subsection{Deposits and Refunds}
 defined later.  Figure~\ref{fig:functions:deposits-refunds} defines the deposit
 and refund functions.
 \begin{itemize}
-\item The function $\fun{deposits}$ returns the total deposits that have to be
+\item The function $\fun{totalDeposits}$ returns the total deposits that have to be
```

Comment:
just `\totalDeposits` here?

Reply:
that can sometimes play weird with spaces, you'd need `\totalDeposits{}{}{}` or however many args. and can leave spaces.

Reply:
ok, didn't think of that